src/PAModelTests/PublicSurfaceTests.cs

```diff
@@ -26,6 +26,7 @@ public void Test()
                 $"{ns}.{nameof(ChecksumMaker)}",
                 $"{ns}.{nameof(ErrorContainer)}",
                 $"{ns}.{nameof(Error)}",
+                $"Microsoft.PowerPlatform.YamlConverter"
```

Comment:
nit: $"Microsoft.PowerPlatform.{nameof(YamlConverter)}"

and add using Microsoft.PowerPlatform;